app/core/db.py

```
@@ -35,7 +40,10 @@ async def __call__(self) -> AsyncEngine:
         """Return an instance of AsyncEngine class."""
 
         if not self.instance:
-            self.instance = create_async_engine(ConfigClass.OPS_DB_URI)
+            try:
+                self.instance = create_async_engine(ConfigClass.OPS_DB_URI)
+            except SQLAlchemyError as exc:
+                logger.exception('Error DB connect', exc_info=exc)
```

It's better to add raise here because the exception will be silenced.